streams/src/main/java/org/apache/kafka/streams/processor/internals/PartitionGroup.java

```diff
@@ -216,8 +216,9 @@ long partitionTimestamp(final TopicPartition partition) {
     }
 
     // creates queues for new partitions, removes old queues, saves cached records for previously assigned partitions
-    void updatePartitions(final Set<TopicPartition> newInputPartitions, final Function<TopicPartition, RecordQueue> recordQueueCreator) {
+    void updatePartitions(final Set<TopicPartition> inputPartitions, final Function<TopicPartition, RecordQueue> recordQueueCreator) {
```

Comment:
@cadonna This is a bug exposed from the unit test now: we should not modify the passed in params directly.